backend/coins/btc/handlers/handlers.go

```diff
@@ -580,3 +583,105 @@ func (handlers *Handlers) postSetTxNote(r *http.Request) (interface{}, error) {
 
 	return nil, handlers.account.SetTxNote(args.InternalTxID, args.Note)
 }
+
+type signingResponse struct {
+	Success bool `json:"success"`
+	Aborted bool `json:"aborted"`
+}
+
+type errorResponse struct {
+	Success      bool   `json:"success"`
+	ErrorMessage string `json:"errorMessage"`
+	ErrorCode    string `json:"errorCode"`
+}
+
```

Comment:
Sure, I'll combine them. 
I didn't think of a user abort as an "error" per se, since nothing went wrong, but a valid signing response with a conscious choice by the user to abort